aea/cli/run.py

```diff
@@ -258,13 +259,27 @@ def _setup_connection(
     default=False,
     help="Install all the dependencies before running the agent.",
 )
+@click.option(
+    "--skip-consistency-check",
+    "skip_check",
+    is_flag=True,
+    required=False,
+    default=False,
+    help="Skip consistency check..",
+)
 @pass_context
 def run(
-    click_context, connection_ids: List[PublicId], env_file: str, is_install_deps: bool
+    click_context,
+    connection_ids: List[PublicId],
+    env_file: str,
+    is_install_deps: bool,
+    skip_check: bool,
 ):
     """Run the agent."""
     ctx = cast(Context, click_context.obj)
     try_to_load_agent_config(ctx)
```

Comment:
I'd prefer a solution where `try_to_load_agent_config` and `validate_config_consistency` are by default run on each `aea` subcommand. Then, also apply the `skip_consistency_check` at the `aea` level so it can be skipped for each run.